.github/workflows/quality-assurance.yml

```diff
@@ -28,14 +28,12 @@ jobs:
       - name: Cache Rust
         id: cache-rust
         env:
-          cache-version: 2
+          cache-version: 3
```

Comment:
Is this related with the issue ?

Reply:
Can you add a comment to explain what this field is for and when it should be updated ?

I though this was related to selecting the version of the github action ^^